app/Controllers/statsController.php

```diff
@@ -17,6 +17,7 @@ public function firstAction() {
 
 		$this->_csp([
 			'default-src' => "'self'",
+			'img-src' => '* data:',
```

Comment:
I don't understand how this change is related to the PR

Reply:
In case the logo points to an image from an HTTP address

Reply:
Yes, because at the moment we do not have the ability to edit the CSP in a finer manner, so we are replacing the full CSP declaration

Reply:
Ah right, I thought default CSP and declared CSP were merged in `declareCspHeader()`!

Reply:
That would be smart but not yet. Adding rules to make the policies *stricter* is trivial (natively supported by CSP by just adding lines), but merging / relaxing is more difficult